lib/galaxy/config.py

```diff
@@ -1196,3 +1206,15 @@ def _configure_models(self, check_migrate_databases=False, check_migrate_tools=F
     def _configure_signal_handlers(self, handlers):
         for sig, handler in handlers.items():
             signal.signal(sig, handler)
+
+    def _wait_for_database(self, url):
+        from time import sleep
```

Comment:
I'd move the standard library import to the top of the file (line 21).

Reply:
I would too, no question about that! Was trying to be consistent with the rest of the file (and tread softly). So yes, will do.

Reply:
We can do without it: time module is already imported, so just time.sleep(pause).
I'm leaving the sqlalchemy_utils import in the method.

Reply:
> We can do without it: time module is already imported, so just time.sleep(pause).

Surely better!

Reply:
Done. Thank you for reviewing!

Reply:
Correction: not done. I've just noticed your comment (that I missed) on generating config/galaxy.yml.sample. Working on it...